ambry-store/src/main/java/com/github/ambry/store/DiskManager.java

```diff
@@ -170,7 +171,10 @@ void start() throws InterruptedException {
         }
         Thread thread = Utils.newThread("store-startup-" + partitionAndStore.getKey(), () -> {
           try {
-            partitionAndStore.getValue().start();
+            BlobStore store = partitionAndStore.getValue();
+            store.start();
+            // Increase the disk used capacity once the store is successfully started
+            disk.addUsedCapacityInBytes(store.getReplicaId().getCapacityInBytes());
```

Comment:
A stopped replica would also take some space on disk, we should add the used capacity for stopped replicas as well.

Reply:
Sure. Changed to modify the disk usage in constructor so that it will apply for all replicas that are present on the node.